### PR TITLE
Document FXIOS- #11442 [v104] Fixing missing Parameters Documentation

### PR DESCRIPTION
**Pull request by byaruhaf**
The PR will fix missing Parameters in some function's Documentation

### CODE REVIEW

**Comment by lmarceau** [comment]
Linking issue #11442 and JIRA ticket https://mozilla-hub.atlassian.net/browse/FXIOS-4652

File: Client/Frontend/Home/Wallpapers/Legacy/LegacyWallpaperDataManager.swift [hunk 1]
```diff
@@ -28,11 +28,12 @@ private struct LegacyWallpaperCollection {
     /// region or time limited.
     ///
     /// - Parameters:
-    ///   - names: An array of the names of the wallpapers included in the collection.
+    ///   - wallpaperFileNames: An array of the names of the wallpapers included in the collection.
     ///   - type: The collection type.
-    ///   - expiryDate: An optional expiry date, on and after which the wallpapers in
     ///         the array are no longer shown. If May 1, 2022, the collection is no
     ///         longer visible on May 1, 2022.
```
Comment (lmarceau) [inline Client/Frontend/Home/Wallpapers/Legacy/LegacyWallpaperDataManager.swift:34]: Those lines were part of the `expiryDate` explanation. Should be removed I think
```suggestion
```

File: Client/Frontend/Home/TopSites/DataManagement/TopSitesManager.swift [hunk 2]
```diff
@@ -197,6 +197,8 @@ private extension Array where Element == Site {
     /// Add sponsored tiles to the top sites.
     /// - Parameters:
     ///   - sponsoredTileSpaces: The number of spaces available for sponsored tiles
+    ///   - contiles:An array of Contiles a type of tiles belonging in the Shortcuts section on the Firefox home page.
```
Comment (lmarceau) [inline Client/Frontend/Home/TopSites/DataManagement/TopSitesManager.swift:200]: nit
```suggestion
    ///   - contiles: An array of Contiles a type of tiles belonging in the Shortcuts section on the Firefox home page.
```

File: Client/Frontend/Browser/TabMetadataManager.swift [hunk 3]
```diff
@@ -166,6 +166,7 @@ class TabMetadataManager {
     /// if the title isEmpty we don't record because title can be overriden
     /// - Parameters:
     ///   - url: Site URL from webview
```
Comment (lmarceau) [inline Client/Frontend/Browser/TabMetadataManager.swift:168]: Url isn't a param there anymore it seems
```suggestion
```

Comment (byaruhaf) [reply Client/Frontend/Browser/TabMetadataManager.swift:168]: Yap url was removed

**Comment by byaruhaf** [comment]
applied the changes and expanded the expiryDate explanation

**Comment by lmarceau** [comment]
@byaruhaf could you fix the conflicts in TopSitesManager.swift? Either by merging main in your branch or rebasing on main.

**Comment by byaruhaf** [comment]
@lmarceau done a rebase on main.

**Comment by lmarceau** [comment]
Something went wrong, I see changes and commits that shouldn't be in this PR. Could you fix it 🙏
Screen Shot 2022-07-28 at 5 07 45 PM

Screen Shot 2022-07-28 at 5 07 51 PM

**Comment by byaruhaf** [comment]
@lmarceau going to close this and try again my apologies

**Pull request closed without merging**